integrationTests/vm/esdt/esdtProcess_test.go

```diff
@@ -1208,11 +1341,25 @@ func getESDTTokenData(
 
 	tokenKey := []byte(core.ElrondProtectedKeyPrefix + "esdt" + tokenName)
 	esdtData, err := getESDTDataFromKey(userAcc, tokenKey)
-	assert.Nil(t, err)
+	require.Nil(t, err)
 
 	return esdtData
 }
 
+func checkAddressHasESDTTokensInt64(
```

Comment:
maybe add a TODO to remove the old `checkAddressHasESDTTokens` function?

Reply:
Made some simplifications, now there's only one, using `int64`.